spec/features/change_notes_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+RSpec.feature "Change notes" do
+  scenario "User updates change notes and update type" do
+    given_there_is_a_document
+    when_i_go_to_edit_the_document
+    and_i_fill_in_the_change_note_and_update_type
+
+    then_the_summary_page_displays_change_note_and_update_type
+    and_the_form_displays_the_change_note_and_update_type
+    and_the_publishing_api_should_know_the_update_type_and_change_note
+
+    when_i_publish_the_document
+    then_the_change_note_and_update_type_should_be_cleared_for_the_next_edition
+  end
+
+  def given_there_is_a_document
+    @document = create(:document)
+  end
+
+  def when_i_go_to_edit_the_document
+    visit edit_document_path(@document)
+  end
+
+  def and_i_fill_in_the_change_note_and_update_type
+    @update_request = stub_any_publishing_api_put_content
+
+    fill_in "document[change_note]", with: "Updated banana pricing"
+    choose I18n.t("documents.edit.update_type.minor_name")
+
+    click_on "Save"
+  end
+
+  def then_the_summary_page_displays_change_note_and_update_type
+    expect(page).to have_content "Updated banana pricing"
+  end
+
+  def and_the_form_displays_the_change_note_and_update_type
+    visit edit_document_path(@document)
+    expect(page).to have_selector("textarea[name='document[change_note]']", text: "Updated banana pricing")
+    expect(page).to have_selector("input[name='document[update_type]'][checked='checked'][value='minor']")
```

Comment:
Might be a bit cleaner for these two to be something like `expect(find("document[change_note]")). to have_text` and similar

Reply:
How would you do that for a radio button? This was the only way I found to see if a radio button is checked.

Reply:
I may be totally wrong here because all I've done is google but I think you could do `expect(find("document[update_type]")).to be_checked` since that's a boolean we can use on a field: https://www.rubydoc.info/github/jnicklas/capybara/Capybara%2FNode%2FElement:checked%3F

Reply:
Ah! Nice. Much better now.